src/thor/dao/release_dao.py

```diff
@@ -179,3 +240,28 @@ def get_release_keys():
 
     with session_scope() as session:
         return [release.release_id for release in session.query(Release)]
+
+
+class look_upper:

```

Comment:
Maybe we can tweak this name a bit for something more explicit (meaningful in relation to its purpose)...
Do we actually need this function to be placed in a whole new class?
Why can't `release_id_lookup` just join the other class'less functions in this same script?

Reply:
The issue here is that we need to mock the release_id_lookup for the test_write_no_prior test inside test_jjm_write_task_result, and I can't patch the method unless it's associated with a class. Had I known how much trouble this would cause, I might have created the DAOs as classes to be instantiated, but that's a topic for refactoring. 

I will go back and rename the class to make it more meaningful.

Reply:
> The issue here is that we need to mock the release_id_lookup for the test_write_no_prior test inside test_jjm_write_task_result, and I can't patch the method unless it's associated with a class.

Ah. gotcha. Understood. That makes sense.
I think there's a way to patch without placing it in a class, but that's not a big deal anyway. It's ok to move forward with this approach.